test_data_reference.py

```
@@ -0,0 +1,51 @@
+        print(f"{source}: {len(files)} files in {path}")
+        if files and len(files) <= 3:
+            for f in files[:3]:
+                print(f"  - {f.name}")
+    
+    print("\n=== Testing OpenAQ Data ===")
+    latest_jp = manager.get_latest_file('openaq', country='JP')
+    if latest_jp:
+        print(f"Latest Japan OpenAQ file: {latest_jp.name}")
+    
+    print("\n=== Testing OpenMeteo Data ===")
+    start_date = datetime(2023, 1, 1)
```

Ruff (0.12.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)